src/ol/source/tile.js

```diff
@@ -204,6 +204,19 @@ ol.source.Tile.prototype.getTile = function(z, x, y, pixelRatio, projection) {};
 
 
 /**
+ * Get the interim tile most suitable for rendering.
+ * @param {!ol.Tile} tile Most recent tile (head of the interim tile list)
+ * @return {!ol.Tile} Best tile for rendering at this location.
+ *
+ */
+ol.source.Tile.prototype.getInterimTile = function(tile) {
```

Comment:
This was done for the benefit of the renderer that deals with a ol.source.Tile, not an ol.source.TileImage as far as I understand. It makes sense to move the getInterimTile to the ol.TileImage, but we would then also need a method on ol.Tile matching the method you commented on. I'll try it out and you can see what you think when I update the pull request, perhaps I am misunderstanding something.